visualization_tool/docs/USAGE.md

```diff
@@ -45,10 +45,20 @@ The tool provides several options to filter the resources. You can filter the re
 
 #### Searching for resources
 
+We have two convenient methods to search for resources:
+
+- Normal Search
+
 You can search for resources by their name in the search bar. The tool will show the resources that contain the search term in their name.
 
 ![GCP Scanner Visualizer](../../misc/visualization_tool_images/search_resources.webp)
 
+- Regular Expression Search
+
+You can search for resources by providing a valid regular expression of the resource name in the search bar. The tool will show the resources that matches the expression.
```

Comment:
Thanks for adding that. Sorry for not providing that in the original request but could you also please add a link referencing documentation of that regular expression format? Because it might not be clear on what tokens are acceptable for end user.

Reply:
Added the link to reference.